concepts/type-conversion/introduction.md

```diff
@@ -1,3 +1,25 @@
-# Introduction
+# Introduction to Type Conversion
 
-TODO: add introduction for type-conversion concept
+In Go, assignment of a value between different types requires explicit conversion.
+Converting between types is done via a function with the name of the type to convert to.
+A value `v` is converted to type `T` using `T(v)`.
```

Comment:
This line is nice for the about file but could probably be omitted since there is the text above and the example below in the introduction already.
```suggestion
```